src/kata2.booleanToWord.js

```diff
@@ -1,3 +1,7 @@
-const booleanToWord = (boolean) => {};
-
+const booleanToWord = (boolean) => {
+  if (boolean) {
+    return "Yes";
+  }
+  return "No";
+};
```

Comment:
You can refactor this a little to make it into a one liner using a ternary operator:
```
const booleanToWord = (boolean) => (boolean === true ? "Yes" : "No");
```